apps/angular/crud/src/app/components/item.component.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Todo } from '../model/todo.interface';
+
+@Component({
+  selector: 'app-item',
+  standalone: true,
+  imports: [CommonModule],
+  template: `
+    <div>
+      <span aria-label="title">
+        {{ item.title }}
+      </span>
+      <button
+        aria-label="update button"
+        [disabled]="disabledTodosIds?.includes(item.id)"
+        (click)="update(item)">
+        Update
+      </button>
+      <button
+        aria-label="delete button"
+        [disabled]="disabledTodosIds?.includes(item.id)"
+        (click)="delete(item)">
+        Delete
+      </button>
+      <ng-container *ngIf="errorTodosIds?.includes(item.id)">
+        Error occuried
+      </ng-container>
+    </div>
```

Comment:
next step: loading should be handle per item as well.

Reply:
can you please provide a little more? You suggest create some inner store for each item?